### ISSUE TITLE
Rule set-up to prevent overwrites

### ISSUE THREAD

**Issue by jmabad100** [rdmtc/node-red-contrib-sun-position#137]
Hi, 

Does somebody knows how to set-up a rule with a condition to prevent an overwrites?

Thank you so much.

**Comment by Hypnos3** [comment]
That is not possible. An override is(currenlty) always evaluated with an higer priority than any rule. Multiple overrides could have a priority.

Maybe a feature request to give the rules also a priority property.

What is currently possible is to work completely without overrides and only use rules. Rules can also access msg properties and go active accordingly or not.

**Comment by jmabad100** [comment]
Thank Hypnos3,

I was just asking, because the documentation says:

**_There are a special configuration for rules with a condition, with which it can be prevented to allow overrides._**

And I was not able to find this set-up.

Thanks

**Comment by Hypnos3** [comment]
Sorry that the documentation here is misleading.

Unfortunately I don't know what I wanted to say with this sentence in the documentation. I will change it in the next remease.

**Comment by jmabad100** [comment]
Don’t worry

Thank you so much for everything.

**Comment by Hypnos3** [comment]
In the next release a "importance" (priority is now renamed to importance) for rules is implemented, where can be setup a rule with a higher "importance" than an override.

This is still under test. If you feel brave you can already install it from command line from the node-red directory with (standard Linux):
```bash
npm install node-red-contrib-sun-position@1.1.0-alpha
```

If you are using a Homematic CCU the commands are (to use from ssh login):
```bash
source /usr/local/addons/redmatic/home/.profile
cd /usr/local/addons/redmatic/var
npm install --save --no-package-lock --global-style --save-prefix="~" --production node-red-contrib-sun-position@1.1.0-alpha
```

**Comment by jmabad100** [comment]
> In the next release a "importance" (priority is now renamed to importance) for rules is implemented, where can be setup a rule with a higher "importance" than an override.
> 
> This is still under test. If you feel brave you can already install it from command line from the node-red directory with (standard Linux):
> 
> ```shell
> npm install node-red-contrib-sun-position@1.1.0-alpha
> ```
> 
> If you are using a Homematic CCU the commands are (to use from ssh login):
> 
> ```shell
> source /usr/local/addons/redmatic/home/.profile
> cd /usr/local/addons/redmatic/var
> npm install --save --no-package-lock --global-style --save-prefix="~" --production node-red-contrib-sun-position@1.1.0-alpha
> ```

Hi hypnos3

I have installed version 1.1.0 alpha using npm, but it is not updating on node red. Version 1.0.17 continues to appear. What should I do to work with version 1.1.0 alpha.

Thanks

**Comment by Hypnos3** [comment]
Today I have released Version 1.1.0. It should be available in the Standard Node-Red package manager.
